### DISCUSSION TITLE
Wine 8

### DISCUSSION THREAD

**Discussion by nicks-boson** [LinSoftWin/Photoshop-CC2022-Linux#90] [Q&A]
Will photoshop run better on Wine 8?

**Comment by MiMillieuh** [comment]
It runs a bit better with Nvidia GPUS but nothing has changed with AMD gpus.

**Comment by rafaelmarrichi** [comment]
instalei com o wine 8.1... porem quando vou criar um novo documento, aparece a msg:  não foi possivel completar o seu pedido devido a um erro de programa
alguem ja passou por isso ?

**Reply by MiMillieuh** [reply]
Can I have the logs when you run the app with the terminal

And please ask your question in English I don't understand...
